indico/modules/events/abstracts/models/reviews.py

```diff
@@ -137,3 +139,9 @@ class AbstractReview(db.Model):
     @return_ascii
     def __repr__(self):
         return format_repr(self, 'id', 'abstract_id', 'user_id', proposed_action=None)
+
+    @property
+    def score(self):
+        if not self.ratings:
+            return None
+        return sum((x.value for x in self.ratings)) / len(self.ratings)
```

Comment:
Not sure if this logic is correct. We may need to take the scale into account. If you have a scale from 1 to 5 we want the score in that range.

What we store in the rating is not the value on the scale (that's often an ugly float depending on the scale) but the index of the selected option (since you could change the scale after there are already reviews and we don't want to have to re-calculate all those values in the DB).